content/en/blog/2023/http-conventions-declared-stable.md

```diff
@@ -0,0 +1,264 @@
+---
+title: HTTP semantic conventions are declared stable
+linkTitle: HTTP semantic conventions
+date: 2023-11-03
+author: '[Trask Stalnaker](https://github.com/trask) (Microsoft)'
+cSpell:ignore: stalnaker trask
+---
+
+Earlier this year we embarked on a three month project to stabilize HTTP semantic conventions.
+
+We are proud to announce (nine months later), that the HTTP semantic conventions are now the first OpenTelemetry semantic convention to be declared stable!
+
+This stable version is a significant evolution over the previous unstable version.
+
+Because of the number of changes, and because of the number of users impacted by these changes, we have mapped out
+a transition plan for instrumentations to follow when updating to the stable version which prioritizes our users.
+
+We also want to share some of the reasons behind these changes, as we think that the stable version brings many
+significant improvements which will make the pain of this upgrade worth it in the long run.
+
+## Aligning with other open standards
+
+Earlier this year we announced [the merging of Elastic Common Schema into OpenTelemetry Semantic Conventions]([https://opentelemetry.io/blog/2023/ecs-otel-semconv-convergence/]).
+
+This merger brought several direct benefits to the HTTP semantic conventions.
+
+One was the transition from `net.host.*` and `net.peer.*`
+to `client.*` and `server.*`, which has a number of benefits:
+* works better for logs (where there's no span kind)
+* improves correlation across client and server telemetry (e.g. since you can join on `server.address` instead of joining where
+`net.peer.addr` == `net.host.addr`)
+* is generally more intuitive compared to `net.host.*` and `net.peer.*`
+* left open the `network.peer.*` and `network.local.*` namespace for low-level network instrumentation where peer and local concepts are more natural
+
+Another was the `url.*` namespace, which allows these attributes to be reused in non-HTTP semantic conventions
+in the future.
+
+And another was improved consistency of namespaces,
+e.g. `http.request.method` instead of `http.method`, and `http.response.status_code` instead of `http.status_code`,
+which match the other `http.request.*` and `http.response.*` attributes.
+
+Better alignment with Prometheus by adopting seconds as the standard unit for metrics.
```

Comment:
it does not seem to be relevant to ECS, perhaps `Metric cardinality` below can be renamed to something like `Metrics improvements` and this sentence can go there?